packages/react-components/src/button/src/Button.css

```diff
@@ -182,162 +202,163 @@ a.o-ui-button {
 /* OUTLINE | ACTIVE */
 .o-ui-button-outline:active,
 .o-ui-button-outline.o-ui-button-active {
-    box-shadow: 0 0 0 1px var(--marine-500) inset;
-    color: var(--white);
-    background-color: var(--marine-500);
-}
-
-/* OUTLINE | ACTIVE | FOCUS */
-.o-ui-button-outline.o-ui-button-focus:active,
-.o-ui-button-outline.o-ui-button-focus.o-ui-button-active,
-.o-ui-button-outline:focus:active,
-.o-ui-button-outline:focus.o-ui-button-active {
-    box-shadow: 0 0 0 1px var(--marine-500) inset, var(--o-ui-focus-ring);
+    box-shadow: 0 0 0 1px var(--quaternary-border) inset;
+    color: var(--static-white);
+    background-color: var(--foreground-color-5);
 }
 
 /* OUTLINE | ACTIVE | DISABLED */
 .o-ui-button-outline[disabled]:active,
 .o-ui-button-outline[disabled].o-ui-button-active,
 .o-ui-button-outline.o-ui-button-disabled:active,
 .o-ui-button-outline.o-ui-button-disabled.o-ui-button-active {
-    box-shadow: 0 0 0 1px var(--marine-500) inset;
-    color: var(--marine-500);
-    background-color: var(--white);
+    box-shadow: 0 0 0 1px var(--quaternary-border) inset;
+    color: var(--secondary-text);
+    background-color: var(--foreground-color-1);
 }
 
 /* OUTLINE | PRIMARY */
 .o-ui-button-outline.o-ui-button-primary {
-    box-shadow: 0 0 0 1px var(--primary-500) inset;
-    color: var(--primary-500);
+    box-shadow: 0 0 0 1px var(--brand-border) inset;
```

Comment:
I guess the weird thing is that to me "primary" always meant the brand color. Why does it change now?

Reply:
Also, it creates a discrepancy between how Tachyons is structured:

![image](https://user-images.githubusercontent.com/794579/104755470-764d0700-5728-11eb-8eea-f0c28559dc76.png)
![image](https://user-images.githubusercontent.com/794579/104755513-84028c80-5728-11eb-8363-87d25efdb2ae.png)

As you can see in the screenshots, tachyons does handle primary as a product and brand as "marine", "moonstone" etc..